src/libraries/System.IO.Hashing/src/System/IO/Hashing/Crc32.cs

```diff
@@ -77,6 +77,11 @@ protected override void GetHashAndResetCore(Span<byte> destination)
             _crc = InitialState;
         }
 
+        /// <summary>Gets the current computed hash value without modifying accumulated state.</summary>
+        /// <returns>The hash value for the data already provided.</returns>
```

Comment:
I think I'd expect the endianness to be here in a remark(s). Something like

```suggestion
        /// <returns>The hash value for the data already provided.</returns>
        /// <remarks>The return value from this method is the little-endian interpretation of <see cref="GetCurrentHash"/>.</remarks>
```

Reply:
This is the numerical result of the algorithm; we're not returning or formatting bytes from a number here.  If anything, GetCurrentHash is the little-endian output of this... but you asked me explicitly _not_ to include endianness discussion in the method remarks. :smile:

Reply:
In the byte-returning ones, yes.  Because the fundamental basis of this API is that it produces bytes (XxHash128 won't have an integral output on NS2.0, after all).   Since everything has byte-based output (and the base class enforces that), the endianness comment belongs (to me) on the new variants.

Reply:
I think that's confusing.  If all you do is type:
```C#
ulong hash = XxHash3.HashToUInt64(bytes);
```
you're not thinking of this in terms of the polymorphic API design nor the virtual base GetCurrentHash method.  And even if you do, having a reference in this static method to some other instance method makes you have to understand what that other instance method is doing.  And then you need to understand from an algorithm perspective that that method is doing a machine-to-little-endian conversion that this method is then undoing.

Reply:
Well, the seealso's would be instance->instance or static->static, but yeah.  If you think it's confusing, we can leave the endianness remark and/or the seealsos out.